python/xoscar/collective/tests/test_pygloo.py

```diff
@@ -22,6 +22,10 @@
 
 system_name = platform.system()
 
+# When running tests on Windows, some tests may fail due to file system
+# permission issues. Therefore, we skip the relevant tests related to
+# filestore on Windows.
+
```

Comment:
You can use ``pytest.skip`` to skip some test on windows, and add comments there.

Reply:
There will be some error on windows when testing `filestore` on windows, so I just run `test_pygloo.py` on `unix`